resources/assets/css/_general.css

```diff
@@ -9,62 +9,34 @@
 
 /* Navbar container */
 .navbar-container {
-    @apply px-8;
-}
-
-@screen md {
-    .navbar-container {
-        @apply px-10;
-    }
+    @apply px-8 md:px-10;
 }
 
 /** Headings with H **/
 h1,
 h2,
 h3,
 h4 {
-    @apply font-bold text-theme-secondary-900;
-}
-
-h1 {
-    @apply font-bold;
-}
-
-.dark h1,
-.dark h2,
-.dark h3,
-.dark h4 {
-    @apply text-theme-secondary-200;
+    @apply font-bold text-theme-secondary-900 dark:text-theme-secondary-200 leading-tight;
```

Comment:
additional `font-bold` on h1 was unnecessary. also `.dark h1` can be replaced with `dark:text-theme...`